gp2gp-translator/src/main/java/uk/nhs/adaptors/pss/translator/task/MhsQueueMessageHandler.java

```diff
@@ -37,6 +36,7 @@ public class MhsQueueMessageHandler {
     private final MDCService mdcService;
     private final EhrExtractMessageHandler ehrExtractMessageHandler;
     private final AcknowledgmentMessageHandler acknowledgmentMessageHandler;
+    private final AttachmentMessageHandler continueMessageHandler;
```

Comment:
Make sure this is refactored in with the attachment to continue class name change.